tests/integration/gemm/int4_dequantization/main.cpp

```diff
@@ -206,27 +206,28 @@ void dequantize_gemm_run(int iter) {
     static constexpr uint32_t prefetch_distance = 3;
 
     using mem_desc_a_t = xetla::mem_desc_t<data_type_a, mem_layout::row_major,
-            mem_space::global>;
+            mem_space::global, DEVICE_MEM_ALIGNMENT / sizeof(data_type_a)>;
     using mem_desc_b_t = xetla::mem_desc_t<data_type_b, mem_layout::row_major,
-            mem_space::global>;
+            mem_space::global, DEVICE_MEM_ALIGNMENT / sizeof(data_type_b)>;
     using mem_desc_c_t = xetla::mem_desc_t<data_type_c, mem_layout::row_major,
-            mem_space::global>;
+            mem_space::global, DEVICE_MEM_ALIGNMENT / sizeof(data_type_c)>;
 
     using compute_attr = xetla::group::compute_attr_t<data_type_acc_in,
             data_type_acc_in, data_type_acc>;
     using perf_tuning_knob = xetla::group::perf_tuning_knob_t<sg_tile_k,
             prefetch_distance, periodic_sync_interval>;
     using compute_policy
             = xetla::group::compute_policy_int4_dequantize_xmx<compute_attr,
-                    perf_tuning_knob, data_type_scale, data_type_zero_pt,
-                    dequant_s, gpu_arch::Xe>;
+                    perf_tuning_knob, gpu::xetla::group::quant_mode::S4_CLIP,
+                    data_type_scale, data_type_zero_pt, dequant_s,
+                    gpu_arch::Dg2>;
     using gemm_t = xetla::group::gemm_t<compute_policy, tile_shape,
             mem_desc_a_t, mem_desc_b_t>;
 
     using epilogue_t = xetla::group::epilogue_t<
-            xetla::group::epilogue_policy_default<gpu_arch::Xe>, tile_shape,
+            xetla::group::epilogue_policy_default<gpu_arch::Dg2>, tile_shape,
```

Comment:
pass gpu_arch as template parameters and determine through SIMD width during runtime to maintain PVC capability

Reply:
Currently you need to know the arch when compiling.

Reply:
I think compute_policy, epilogue_t and other templates should share the same GPU architecture, explicitly giving each gpu_arch may create unexpected bugs.

Reply:
> I think compute_policy, epilogue_t and other templates should share the same GPU architecture, explicitly giving each gpu_arch may create unexpected bugs.

Echo. I also think this is a history debt. Before us, no one enabled other gpu_arch. We can implement some kinds of template expression.  
cc our expert on template @DDEle

Reply:
Regarding the modification of arch, a PR can be specially organized. But this may not be conducive to synchronization with innersource